README.md

```diff
@@ -1166,25 +1166,21 @@ Each spoken language is available as a dedicated npm package (e.g. @picovoice/po
     <script src="https://unpkg.com/@picovoice/porcupine-web-en-worker/dist/iife/index.js"></script>
     <script src="https://unpkg.com/@picovoice/web-voice-processor/dist/iife/index.js"></script>
     <script type="application/javascript">
+      function keywordDetectionCallback(keyword) {
+        console.log("Porcupine detected " + keyword);
```

Comment:
I wonder if this the cool JS way of doing it?

```javascript
`Porcupine detected  {keyword}`
```

I actually am not sure about all implications

Reply:
you are absolutely right